libcontainer/system/linux.go

```diff
@@ -83,8 +83,9 @@ func Setctty() error {
  * Detect whether we are currently running in a user namespace.
  * Copied from github.com/lxc/lxd/shared/util.go
  */
-func RunningInUserNS() bool {
-	file, err := os.Open("/proc/self/uid_map")
+func RunningInUserNS(pid int) bool {
+	path := fmt.Sprintf("/proc/%d/uid_map", pid)
```

Comment:
If you address the above comment, this change is unnecessary (because we only ever use this to find out if *our* process is running in a user namespace).